provisioning/tasks/dashboard.yml

```diff
@@ -0,0 +1,11 @@
+- name: Ensure the dashboard directory exists
+  file:
+    path: "{{ dashboard_install_dir }}"
+    state: directory
+    mode: 0755
```

Comment:
Could you add (for backwards compatibility, and to give people a way to disable it if they want):

```
  when: dashboard_install_dir is defined
```

To both this task and the next? Or if it looks cleaner, inside `provisioning/playbook.yml`.